manual/source/Infrastructure/Contribute/index.rst

```diff
@@ -8,7 +8,7 @@ SeqAn is on GitHub: http://github.com/seqan/seqan
 Use the GitHub page to fork SeqAn, create tickets and/or pull requests.
 You can also follow and like the project there!
 
-Contributing Code or documentation
+Contributing Code or Documentation
 ----------------------------------
 
 If you are unfamiliar with git, you need to learn about it first. TODO link
```

Comment:
hm, they are ok, but something more like a "git in 30min" page would be better I think... but it's not so important.